uint8 game_type       # 比赛类型
uint8 game_progress   # 比赛阶段
uint16 stage_remain_time # 当前阶段剩余时间，单位s
uint64 sync_time_stamp  # 机器人接收到该指令的精确 Unix 时间
builtin_interfaces/Time stamp # 时间戳 